src/answer/numeric.rs

```diff
@@ -31,11 +31,19 @@ pub enum Numeric {
 
 impl Numeric {
     pub fn into_i64(self) -> i64 {
-        if let Self::Long(value) = self { value } else { panic!() }
+        if let Self::Long(value) = self {
```

Comment:
This expansion is definitely uglier than the original - can we configure `rustfmt` to use the original?

Reply:
Technically yes -- this is what `use_small_heuristics = "Max"` does -- but this feature is not available in stable yet.